### PR TITLE
Nuke Op Datumization

### PR DESCRIPTION
**Pull request by AnturK**
This one is half-done since the halfwin stuff needs be translated into objectives 

- [x] Finish Lone Op
- [x] Fully port nukeop win conditions/partial wins into objectives.
- [x] Testing
- [x] More testing

### CODE REVIEW

**Comment by JJRcop** [comment]
Getting stale

**Comment by AnturK** [comment]
I keep getting distracted with other stuff

File: code/datums/antagonists/nukeop.dm [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+	if(nuke_team && nuke_team.syndicate_name)
+		owner.current.real_name = "[nuke_team.syndicate_name] [title]"
+	else
+		owner.current.real_name = "Syndicate [title]"
+
+/datum/antagonist/nukeop/leader/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ops.ogg',100,0)
+	to_chat(owner.current, "<B>You are the Syndicate [title] for this mission. You are responsible for the distribution of telecrystals and your ID is the only one who can open the launch bay doors.</B>")
+	to_chat(owner.current, "<B>If you feel you are not up to this task, give your ID to another operative.</B>")
+	to_chat(owner.current, "<B>In your hand you will find a special item capable of triggering a greater challenge for your team. Examine it carefully and consult with your fellow operatives before activating it.</B>")
```
Comment (Cyberboss) [inline code/datums/antagonists/nukeop.dm]: just to_chat(owner, ... all of these

File: code/datums/antagonists/nukeop.dm [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+	return
+
+/datum/objective_team/nuclear/proc/disk_rescued()
+	for(var/obj/item/disk/nuclear/D in GLOB.poi_list)
+		if(!D.onCentCom())
+			return FALSE
+	return TRUE
+
+/datum/objective_team/nuclear/proc/operatives_dead()
+	for(var/datum/mind/operative_mind in members)
```
Comment (Cyberboss) [inline code/datums/antagonists/nukeop.dm]: typeless

File: code/datums/antagonists/nukeop.dm [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+		if(ishuman(operative_mind.current) && (operative_mind.current.stat != DEAD))
+			return FALSE
+	return TRUE
+
+/datum/objective_team/nuclear/proc/syndies_escaped()
+	var/obj/docking_port/mobile/S = SSshuttle.getShuttle("syndicate")
+	return (S && (S.z == ZLEVEL_CENTCOM || S.z == ZLEVEL_TRANSIT)) ? 1 : 0
```
Comment (Cyberboss) [inline code/datums/antagonists/nukeop.dm]: TRUE FALSE

File: code/datums/antagonists/nukeop.dm [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+			to_chat(world, "<FONT size = 3><B>Syndicate Minor Victory!</B></FONT>")
+			to_chat(world, "<B>[syndicate_name] operatives survived the assault but did not achieve the destruction of [station_name()].</B> Next time, don't lose the disk!")
+		else
+			to_chat(world, "<FONT size = 3><B>Neutral Victory</B></FONT>")
+			to_chat(world, "<B>Mission aborted!</B>")
+
+	var/text = "<br><FONT size=3><B>The syndicate operatives were:</B></FONT>"
+	var/purchases = ""
+	var/TC_uses = 0
+	for(var/datum/mind/syndicate in members)
```
Comment (Cyberboss) [inline code/datums/antagonists/nukeop.dm]: ditto?

File: code/datums/antagonists/nukeop.dm [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+		text += SSticker.mode.printplayer(syndicate) //to be moved
+		for(var/datum/component/uplink/H in GLOB.uplinks)
```
Comment (Cyberboss) [inline code/datums/antagonists/nukeop.dm]: ditto

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
Testing seems fine

File: code/datums/antagonists/nukeop.dm (hunk 3 above)
Comment (Cyberboss) [reply code/datums/antagonists/nukeop.dm]: actually this, doesn't need a ternery at all

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**